pinot-core/src/test/java/org/apache/pinot/core/indexsegment/mutable/MutableSegmentImplTest.java

```diff
@@ -139,7 +145,9 @@ public void testDataSourceForSVColumns() {
 
           int actualDictId = actualSVIterator.nextIntVal();
           int expectedDictId = expectedSVIterator.nextIntVal();
-          Assert.assertEquals(actualDictionary.get(actualDictId), expectedDictionary.get(expectedDictId));
+          // Only allow the default segment name to be different
+          if (!column.equals(CommonConstants.Segment.BuiltInVirtualColumn.SEGMENTNAME))
```

Comment:
(nit) add braces for if statement

Reply:
Do you still need this after fixing the segment name?

Reply:
added the braces. I abandoned the segment name change. Hope this will make it a bit easier to read.